lib/evaluate.js

```diff
@@ -228,8 +230,27 @@ export const evaluate = async ({
  * @param {bigint} roundIndex
  * @param {import('./preprocess.js').Measurement[]} measurements
  * @param {import('./typings.js').RoundDetails} sparkRoundDetails
+ * @param {Pick<Console, 'log' | 'error'>} logger
  */
-export const runFraudDetection = async (roundIndex, measurements, sparkRoundDetails) => {
+export const runFraudDetection = async (roundIndex, measurements, sparkRoundDetails, logger) => {
+  const randomness = await getRandomnessForSparkRound(Number(sparkRoundDetails.startEpoch))
```

Comment:
Let's watch this one in prod and add p-retry if it becomes a problem

Reply:
That's a great point! I did not consider that this external API service could fail 🙈 

Do you know if we have any alert that will post a Slack message when the round evaluation failed?

Reply:
There is no such alert, it would be reported to Sentry :|

Reply:
Hmm, we are rather slow to act on Sentry errors.

Do you know if we have any alerts parsing the logs and posting a Slack message if a log message matches a given pattern? I think that would be the easiest solution.

Reply:
https://www.papertrail.com/help/alerts/

Reply:
I configured Papertrail - Slack integration.

Reply:
See https://github.com/filecoin-station/spark-evaluate/pull/297